src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -3466,6 +3694,16 @@ namespace HVACUnitarySystem {
             // Get the Controlling Zone or Location of the Thermostat
             UnitarySystem(UnitarySysNum).ControlZoneNum = UtilityRoutines::FindItemInList(Alphas(iControlZoneAlphaNum), Zone);
 
+            // check that control zone name is valid for load based control
```

Comment:
Added warning that control zone name was missing when Control Type = Load or SZVAV.